R/features.R

```diff
@@ -12,10 +12,16 @@ validate_extendr_features <- function(features, suppress_warnings) {
     discard_empty()
 
   if (!isTRUE(suppress_warnings) && length(unknown_features) > 0) {
-    cli::cli_warn(c(
-      "Found unknown {.code extendr} feature{?s}: {.val {unknown_features}}.",
-      inf_dev_extendr_used() # nolint: object_usage_linter
```

Comment:
this function printed the line `"Are you using a development version of {.code extendr}?"` and was used only once. I figured it was appropriate to delete it.

Reply:
It was used in two places AFAIK, but you can delete it if it is no longer used.

Reply:
Ope! You're correct. I can revert but remove the `"i" = "....."` portion.

Reply:
Reverted. Put `inf_dev_extendr_used()` into a new `R/utils.R` file. Since `R/ui.R` no longer exists there was no clear home for such a utility function.